File: EnotiQuiz/app/src/main/java/com/enoti/enotiquiz/LogoActivity.java
package com.enoti.enotiquiz;

import android.content.pm.ActivityInfo;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.MotionEvent;
import android.widget.ImageView;

public class LogoActivity extends AppCompatActivity {

    private ImageView logo;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_logo);
        setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
        logo = (ImageView)findViewById(R.id.Logo);
        if (getIntent().getStringExtra("logo").equals("enoti")) { // Получение и высвет нужного логотипа
            logo.setImageDrawable(getResources().getDrawable(R.drawable.enotioriginal));
        }
        else if (getIntent().getStringExtra("logo").equals("nejdan")) {
            logo.setImageDrawable(getResources().getDrawable(R.drawable.nejdanoriginal));
        }
    }

    @Override
    public boolean onTouchEvent(MotionEvent event) { // Нажатие на экран - выход из Activity
        finish();
        return true;
    }
}

File: EnotiMendeleev/app/src/main/java/com/enoti/enotimendeleev/LogoActivity.kt
package com.enoti.enotimendeleev

import android.content.pm.ActivityInfo
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.MotionEvent
import kotlinx.android.synthetic.main.activity_logo.*

class LogoActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_logo)
        requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_PORTRAIT
    }

    override fun onTouchEvent(event: MotionEvent): Boolean { // Нажатие на экран - выход из Activity
        finish()
        return true
    }
}